submissions/omawumiRebeccaMoju/HTML/BLK.html

```diff
@@ -45,17 +45,17 @@ <h3>BLK_My Naturals</h3>
         <p><strong>PRICE: </strong><em>1500.00</em></p> 
         </p>
     </section>
-</header>
+</main>
 <footer>
     <h4>SIMILAR PRODUCTS</h4>
     <section>
         <p>
-            <ul>
+            <ol>
                 <li>BLK_My Naturals Hydrating Serum</li>
                 <li>BLK_My Naturals Curls Activator</li>
                 <li>BLK_My Naturals Nourishing Conditioner</li>
                 <li>BLK_My Naturals Leave-in Conditioner</li>
-            </ul>
+            </ol>
         </p>
     </section>
     <p> &copy; 2020 Online BLK_My Naturals</p>
```

Comment:
Are you sure you want to display your products list in the footer? It's best you still put it within the main tag. @wuhmi